crypto/x509/x_pubkey.c

```diff
@@ -472,3 +472,32 @@ ASN1_BIT_STRING *X509_get0_pubkey_bitstr(const X509 *x)
         return NULL;
     return x->cert_info.key->public_key;
 }
+
+/*
+ * Returns 0 for equal, -1 for a < b or error on a, 1 for a > b or error on b,
```

Comment:
please simplify this function to return equal / nonequal / error or so.

Reply:
Done